kratos/mpi/utilities/distributed_csr_conversion_utilities.h

```diff
@@ -0,0 +1,65 @@
+
+//    |  /           | 
+//    ' /   __| _` | __|  _ \   __| 
+//    . \  |   (   | |   (   |\__ \.
+//   _|\_\_|  \__,_|\__|\___/ ____/ 
+//                   Multi-Physics  
+//
+//  License:		 BSD License 
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Riccardo Rossi
+//
+
+#if !defined(KRATOS_DISTRIBUTED_CSR_CONVERSION_UTILITIES_H_INCLUDED)
+#define  KRATOS_DISTRIBUTED_CSR_CONVERSION_UTILITIES_H_INCLUDED
+
+#include "containers/distributed_csr_matrix.h"
+#include "../../../external_libraries/amgcl/backend/builtin.hpp"
+#include "../../../external_libraries/amgcl/backend/interface.hpp"
+#include "../../../external_libraries/amgcl/mpi/distributed_matrix.hpp"
+#include "../../../external_libraries/amgcl/adapter/zero_copy.hpp"
```

Comment:
```suggestion
#include "amgcl/backend/builtin.hpp"
#include "amgcl/backend/interface.hpp"
#include "amgcl/mpi/distributed_matrix.hpp"
#include "amgcl/adapter/zero_copy.hpp"
```

this should work, see [here](https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/linear_solvers/amgcl_solver.h#L40)

Reply:
yes it will work, i ll use it